kglib/kgcn/learn/learn.py

```diff
@@ -25,15 +25,24 @@
 from kglib.kgcn.learn.loss import loss_ops_preexisting_no_penalty
 from kglib.kgcn.learn.metrics import existence_accuracy
 
+from graph_nets import utils_np
+from graph_nets.graphs import GraphsTuple
+
 
 class KGCNLearner:
     """
     Responsible for running a KGCN model
     """
-    def __init__(self, model, num_processing_steps_tr=10, num_processing_steps_ge=10):
+    def __init__(self, model, num_processing_steps_tr=10, num_processing_steps_ge=10, save_fle="save_model.txt", reload_fle=''):
```

Comment:
Looking at this, I think we instead want to have `save_file` as an argument to the method that does the training, and `load_file` can be a mandatory argument for the method that does the testing. That way, the user can make a single KGCNLearner and train it multiple times, saving to multiple different files.
Adding `load_file` to the test method gives us a guarantee for the test method that there is a file provided.

Reply:
Agree. Adjusted it